tests/parquet_test.py

```diff
@@ -120,3 +120,35 @@ def test_get_datasets(self):
 
         for f in glob.glob('pq_test*'):
             os.remove(f)
+
+    @pytest.mark.optional_parquet
+    def test_against_standard_files(self):
+        datadir = 'resources/parquet-testing'
+        filenames = ['alltypes_plain.parquet',
+                     'alltypes_plain.snappy.parquet',
+                     'delta_byte_array.parquet']
+        columns1 = ['id',
+                    'bool_col',
+                    'tinyint_col',
+                    'smallint_col',
+                    'int_col',
+                    'bigint_col',
+                    'float_col',
+                    'double_col',
+                    'date_string_col',
+                    'string_col']
+        columns2 = ['c_customer_id',
+                    'c_salutation',
+                    'c_first_name',
+                    'c_last_name',
+                    'c_preferred_cust_flag',
+                    'c_birth_country',
+                    'c_login',
+                    'c_email_address',
+                    'c_last_review_date']
+        for basename, ans in zip(filenames, (columns1, columns1, columns2)):
+            filename = os.path.join(datadir, basename)
+            columns = ak.get_datasets(filename, is_parquet=True)
+            self.assertListEqual(columns, ans)
+            # Merely test that read succeeds, do not check output
+            data = ak.read_parquet(filename, datasets=columns)
```

Comment:
Is there a reason that we are choosing not to test the output? It seems the correct output data is stored in that repo under some MD files (example: https://github.com/apache/parquet-testing/blob/master/data/delta_byte_array.md), so we could manually copy that data in, similarly to how the columns are added here.

Reply:
The expected output for that file is in https://github.com/apache/parquet-testing/blob/master/data/delta_byte_array_expect.csv, which is a 1000 line CSV file -- a little too much hard-code in :)

Once `ak.read_parquet` can successfully read the file, it might be a good idea to have it read the expected CSV with `pandas.read_csv` and compare, but today `ak.read_parquet` crashes on that file, so it's hard to write the test.

Reply:
Got a chance to look into this a bit; as far as the byte array file goes, it appears to be `DELTA_BYTE_ARRAY-encoded`, which is something that just is not supported by the installs of Arrow that I (we?) have and I don't think is a limitation with our implementation, but I am disappointed to see all of these server crashes popping up and hope to iron some of those out this week.

For the `alltypes_plain.parquet` file, we are seeing the `timestamp96` column slip by because our check is that the type is an instance of the timestamp Arrow type, which it technically is, so I may add an explicit check to make sure we don't try to read timestamp96 columns, which appear to be able to be read in, but the results are incorrect (of course). The `float_col` is what is making the call to `ak.read_parquet()` not work on this file. I am surprised this is failing, but I will try to get this fixed. I have confirmed that we can read double files that are generated by Pyarrow, but am seeing now that float columns fail for some reason, which is quite surprising to me... 

It will be great to get these files testing to get confidence with the Parquet implementation, seeing this bugs come up is definitely exposing some holes in the current implementation.

Reply:
Opened https://github.com/Bears-R-Us/arkouda/pull/1186 to fix the float reading issue and it now is able to read through the whole `alltypes_plain.parquet` file correctly, but is still reading the timestamp96 column.

Reply:
Good news on #1186!

For the DELTA_BYTE_ARRAY, I didn't realize that was a special encoding with complicated support. I just chose that file because it had some longer and more varied string columns, not because we have a definite need to support the encoding. I wish I had a better way to figure out exactly what encodings and parameters our sample data actually uses, but I haven't determined how to parse that info from the output of `parquet-tools inspect`. 

I would be ok with removing the DELTA_BYTE_ARRAY file, but it might also be good to leave it in to check the error handling... what do you think?

Reply:
Ya, I agree that leaving it in as another error check makes sense; I'll keep you updated as I work on a fix for that.

Reply:
Alright, I've opened https://github.com/Bears-R-Us/arkouda/pull/1188 to catch all Parquet errors which should fix the server crashes that you have seen.

I'd be curious on any input in that approach though, which is just catching all errors and reporting them without really doing anything more advanced to handle the errors more gracefully, but at least it gets around server crashes.